### PR TITLE
fixed 26083: Tile images missing on export/import

### PR DESCRIPTION
**Pull request by alex40724**
See
https://mantis.ilias.de/view.php?id=26083

I added the "how to add this to the export" to the documentation as well:
https://github.com/ILIAS-eLearning/ILIAS/blob/release_5-4/Services/Object/README.md

@mjansenDatabay Please have a look at the forum changes.
@mbecker-databay  Please have a look at the test changes.
@klees, @shecken, @nhaagen  Please have a look at the learning sequence, individual assessment changes.

### CODE REVIEW

File: Modules/Forum/classes/class.ilForumExporter.php [hunk 1]
```diff
@@ -46,7 +46,23 @@ public function getXmlRepresentation($a_entity, $a_schema_version, $a_id)
 		return $xml;
 	}
 
-	/**
+    public function getXmlExportTailDependencies($a_entity, $a_target_release, $a_ids)
```
Comment (mjansenDatabay) [inline Modules/Forum/classes/class.ilForumExporter.php:49]: LGTM in general, but looks like a different code style/indentation compared to the other lines in this file.

Comment (alex40724) [reply Modules/Forum/classes/class.ilForumExporter.php:49]: Yes, it uses spaces. How would you like to have the PR being delivered?

Comment (mjansenDatabay) [reply Modules/Forum/classes/class.ilForumExporter.php:49]: Spaces are okay. Feel free to apply PSR-2 to the whole file.

Option 2) Maybe you could contact me (e.g. via short email) if this has been merged, so I can fix the code formatting afterwards.

Comment (mjansenDatabay) [reply Modules/Forum/classes/class.ilForumExporter.php]: Spaces are okay. Feel free to apply PSR-2 to the whole file.

Option 2) Maybe you could contact me (e.g. via short email) if this has been merged, so I can fix the code formatting afterwards.

Comment (alex40724) [reply Modules/Forum/classes/class.ilForumExporter.php:49]: Actually I stopped to follow this "move the whole file" procedure currently completely. These changes cannot be merged up and down automatically and the whole procedure produces too much manual work.

**Review by klees — APPROVED** [review]

**Review by mjansenDatabay — APPROVED** [review]
Approve!

**Comment by klees** [comment]
Thx, @alex40724!

**Review by mbecker-databay — APPROVED** [review]

**Comment by alex40724** [comment]
Merged manually.

**Comment by PurHur** [comment]
Currently the method `getXmlExportTailDependencies($a_entity, $a_target_release, $a_ids)` is duplicated in the class. ilForumExporter.

**Comment by mjansenDatabay** [comment]
@alex40724 Yes, this breaks the forum exporter, and the generation of the artifacts after using composer.

**Pull request closed without merging**